packages/angular/src/builders/webpack-server/webpack-server.impl.ts

```diff
@@ -0,0 +1,58 @@
+import { BuilderContext, createBuilder } from '@angular-devkit/architect';
+import { JsonObject } from '@angular-devkit/core';
+import type { Schema } from './schema';
+
+import { parseTargetString, joinPathFragments } from '@nrwl/devkit';
+import { Workspaces } from '@nrwl/tao/src/shared/workspace';
+import {
+  DevServerBuilderOptions,
+  serveWebpackBrowser,
+} from '@angular-devkit/build-angular/src/dev-server';
+import { existsSync } from 'fs';
+import { merge } from 'webpack-merge';
+import { normalizeOptions } from './lib';
+
+export function webpackServer(schema: Schema, context: BuilderContext) {
```

Comment:
@vsavkin @FrozenPandaz I know our guides on Nx.dev suggest people use `file-server` for serving an incremental build architecture, but does this builder of our own present an opportunity for us to allow incremental build architecture whilst falling through to Angular's own dev server?